test/Escrow.js

```diff
@@ -310,6 +316,34 @@ contract('Escrow', (accounts) => {
       }
     });
 
+    it('optionally writes out internal url & hash', async () => {
+      try {
+        const toAddress = [accounts[0]];
+        const initialStatus = await Escrow.getStatus.call();
+        assert.equal(initialStatus, 0);
+
+        // Transer funds to escrow
+        await HMT.transfer(Escrow.address, 200, { from: accounts[0] });
+
+        // Setup escrow
+        await Escrow.setup(reputationOracle, recordingOracle, 10, 10, recOIpns, repOIpns, url, hash, { from: accounts[0] });
+        const setupStatus = await Escrow.getStatus.call();
+        assert.equal(setupStatus, 1);
+
+        const amountToPay = [4];
+        assert(await Escrow.getFinalResultsUrl.call() === '' && await Escrow.getFinalResultsHash.call() === '',
+              'hash and url should be empty');
+        await Escrow.bulkPayOut(toAddress, amountToPay, url, hash, '000', { from: reputationOracle });
+        assert(await Escrow.getFinalResultsUrl.call() === url && await Escrow.getFinalResultsHash.call() === hash,
+              'either hash or url not set');
+        await Escrow.bulkPayOut(toAddress, amountToPay, '', '', '000', { from: reputationOracle });
+        assert(await Escrow.getFinalResultsUrl.call() === url && await Escrow.getFinalResultsHash.call() === hash,
+                    'hash or url changed when it should not have');
+      } catch (ex) {
+        assert(false);
```

Comment:
does this even need to be caught?

Reply:
no, let's remove this next release. i believe i changed this in an audit branch, we can merge that in at some point

Reply:
Audit stuff wasn't merged separately?

Reply:
No. We wanted to hold off on that if I recall.